### PR TITLE
Add new command for adding users.

### PR DESCRIPTION
**Pull request by boehlke**
Required to add users from command line...

### CODE REVIEW

File: openslides/core/management/commands/createopenslidesuser.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
```
Comment (normanjaeckel) [inline openslides/core/management/commands/createopenslidesuser.py]: Please move this to the users app.

File: openslides/core/management/commands/createopenslidesuser.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+class Command(BaseCommand):
+    """
+    Command to create a new OpenSlides user
+    """
+    help = 'Creates an OpenSlides user.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'first_name',
+            help='the first name of the new user.'
```
Comment (normanjaeckel) [inline openslides/core/management/commands/createopenslidesuser.py]: Typo. First letter should be a capital letter.

File: openslides/core/management/commands/createopenslidesuser.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            'password',
+            help='The password of the new user.'
+        )
+
+    def handle(self, *args, **options):
+        user_data = {
+            'first_name': options['first_name'],
+            'last_name': options['last_name'],
+            'username': options['username'],
+            'password': options['password'],
```
Comment (normanjaeckel) [inline openslides/core/management/commands/createopenslidesuser.py]: The password has to be hashed using django.contrib.auth.hashers.make_password. Or you use a set_password method.

File: openslides/core/management/commands/createopenslidesuser.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'first_name',
+            help='the first name of the new user.'
+        )
+        parser.add_argument(
+            'last_name',
+            help='The last name of the new user.'
+        )
+        parser.add_argument(
```
Comment (normanjaeckel) [inline openslides/core/management/commands/createopenslidesuser.py]: The username can be generated using User.objects.generate_username().

**Comment by emanuelschuetze** [comment]
@boehlke travis still failed...
```
$ flake8 openslides tests
openslides/users/management/commands/createopenslidesuser.py:3:1: F401 'openslides.users.models.User' imported but unused
```

**Pull request merged**